swift/test/Ice/scope/Test.ice

```diff
@@ -156,26 +156,54 @@ module Test
 ["swift:module:Test:Inner"]
 module Inner
 {
+    module Test::Inner2
+    {
+        interface I
+        {
+            Test::S opS(Test::S s1, out Test::S s2);
+            Test::SSeq opSSeq(Test::SSeq s1, out Test::SSeq s2);
+            Test::SMap opSMap(Test::SMap s1, out Test::SMap s2);
 
-module Test
-{
+            Test::C opC(Test::C c1, out Test::C c2);
+            Test::CSeq opCSeq(Test::CSeq c1, out Test::CSeq c2);
+            Test::CMap opCMap(Test::CMap c1, out Test::CMap c2);
+
+            void shutdown();
+        }
+    }
+}
 
-module Inner2
+// Check that it's okay for 'swift:module' to have different 'prefix' arguments.
+["swift:module:Test:Different"]
+module Inner
 {
-    interface I
+    class Class1
     {
-        Test::S opS(Test::S s1, out Test::S s2);
-        Test::SSeq opSSeq(Test::SSeq s1, out Test::SSeq s2);
-        Test::SMap opSMap(Test::SMap s1, out Test::SMap s2);
-
-        Test::C opC(Test::C c1, out Test::C c2);
-        Test::CSeq opCSeq(Test::CSeq c1, out Test::CSeq c2);
-        Test::CMap opCMap(Test::CMap c1, out Test::CMap c2);
+        long l;
+    }
 
-        void shutdown();
+    interface I
```

Comment:
We should definitely never name an interface 'I'.

Reply:
I was just being consistent with the rest of this test.
I updated the `scope` tests to use better names.